tests/test_structures.py

```diff
@@ -0,0 +1,79 @@
+# coding: utf-8
+import pytest
+
+from requests.structures import CaseInsensitiveDict, LookupDict
+
+
+class TestCaseInsensitiveDict:
+
+    @pytest.fixture(autouse=True)
+    def setup(self):
+        """
+        CaseInsensitiveDict instance with "Accept" header.
+        """
+        self.case_insensitive_dict = CaseInsensitiveDict()
+        self.case_insensitive_dict['Accept'] = 'application/json'
```

Comment:
Ah, sorry @Stranger6667, but I think @sigmavirus24 wanted [something more like this](https://pytest.org/latest/xunit_setup.html#method-and-function-level-setup-teardown).